app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java

```diff
@@ -95,15 +95,15 @@
     private ArrayAdapter<String> loanPurposeAdapter;
 
     private LoanTemplate loanTemplate;
-    private DialogFragment mfDatePicker;
+    private DialogFragment mfDatePickerDisbursement;
+    private DialogFragment mfDatePickerSubmission;
```

Comment:
Remove this

Reply:
And what should I use to replace them? They are being used... Should I use a `mfDatePicker` for both cases?

Reply:
> And what should I use to replace them? They are being used... Should I use a `mfDatePicker` for both cases?

^^^^

@miPlodder